src/title.js

```diff
@@ -0,0 +1,43 @@
+// LICENSE : MIT
+/*
+ * Gibook usage:
+ *
+ *      [import,title](path/to/file)
+ *      [import,title:<the title>](path/to/file)
+ */
+
+const spaces = "[ \t]*"; // h spaces
+const titleFormat = "([^'\"]*)?";
+
+/*
+ * format: [import,title:<the title>](path/to/file)
+ * Get the specified <the title>
+ * @example:
+ *     parseTitle(label,fileName)
+ * @param {string} label
+ * @param {string} fileName
+ *
+ * @return {boolean,string}
+ */
+export function parseTitle(label, fileName) {
+    let title=[false, undefined];
```

Comment:
Tuple mixed data type is not major in JavaScript.
It will be better that separate `hasTitle` (`regExp.test(string)` is useful) and `parseTitle`?